[DOC] Tempo DS: Clone a provisioned data source doc (#110114)
grafana/grafana: docs/sources/datasources/tempo/configure-tempo-data-source.md

      streamingEnabled:
        search: true
```

### Clone a provisioned data source for Grafana Cloud

If you have a data source that is provisioned by a configuration file in Grafana Cloud, you can clone that provisioned data source and then edit the new data source in the Grafana UI.

For example, let's say you want to edit the **Trace to log**s settings in your Tempo data source that is provisioned on Grafana Cloud.
You'd like to enable traceID and spanID but you can't because the data source is provisioned.

> You could enable these capabilities in a cloned data source.

Author: ```suggestion
By cloning the data source, you'd be able to edit these capabilities.
```
